packages/video_player/video_player_web/lib/src/video_player.dart

```diff
@@ -188,9 +189,40 @@ class VideoPlayer {
     return Duration(milliseconds: (_videoElement.currentTime * 1000).round());
   }
 
+  /// Sets control options
+  Future<void> setControls(VideoPlayerWebOptions controls) async {
+    if (controls.controlsEnabled) {
+      _videoElement.controls = true;
+      final List<String> attributes = <String>[];
+      if (!controls.allowDownload) {
+        attributes.add('nodownload');
+      }
+      if (!controls.allowFullscreen) {
+        attributes.add('nofullscreen');
+      }
+      if (!controls.allowPlaybackRate) {
+        attributes.add('noplaybackrate');
+      }
```

Comment:
Should all this conversion logic live in the `VideoPlayerWebOptions`, and exposed through a `String get controlsList` getter?